.gitignore

```diff
@@ -16,6 +16,9 @@ downloads/
 _trial_temp*/
 docs/notes.txt
 twisted/plugins/dropin.cache
+twistd.log
+vumi_http_proxy/twistd.pid
+vumi_http_proxy/twistd.log
```

Comment:
I think we can drop `vumi_http_proxy/twistd.pid` and `vumi_http_proxy/twistd.log` from the ignore list because those would only be created there if one ran `twistd` from inside the `vumi_http_proxy/`, which seems like something that one shouldn't usually do?